[FIX] default_warehouse_from_sale_team: remove incorrect record rule T#79454

luisg123v opened the pull request:
There was a rule to grant read-only access to pickings. However, the correct way to grant read-only access to a model is to uncheck the "Read" field on the desired rule, which would cause that rule to not be considered when verifying accesses.

ajteposte reviewed (approved):
Looks good to me. c:
 :+1:

luisg123v committed:
[FIX] default_warehouse_from_sale_team: remove incorrect record rule

There was a rule to grant read-only access to pickings. However, the
correct way to grant read-only access to a model is to uncheck the
"Read" field on the desired rule, which would cause that rule to not be
considered when verifying accesses.

isaako34 commented:
LGTM 👍

luisg123v commented:
Thanks a lot @TeposteAJ @isaako34.
